concepts/observability-strategy/strategy.md

```diff
@@ -0,0 +1,77 @@
+
+With that, the goal of the kyma observability should be the seemless enablement of the stages on the left, allowing the ingestion of the signals into any backend system, opening up plenty usage scenarios. Kyma should provide blueprints for the stages on the right, especially focussed on the SAP ecosystem enablement, in order to have at least some guidance for these aspects.
+To sum it up, the goals of kyma observability should be:
+1. provide guides on how to do instrumentation (with potential helpers and auto-instrumentation options)
+2. collect resulting signals instantly when guides are followed, provide customization options for border cases
+3. ship the signals reliable to a configurable vendor-neutral destination
+4. provide blueprints for integration to specific vendors
+
+# Architecture
+
+The idea of the proposal is to introduce a new preconfigured agent layer that's responsible for collecting all telemetry data. Hereby, the collection is very dependent on the signal type itself. These agents can be configured at runtime with different signal pipelines with basic filtering (inclusion and exclusion of signals) and outputs, so that the agents start shipping the signals via the pipelines to the configured backends. The dynamic configuration and management of the agent is handled by a new operator, which is configured using Kubernetes resources. The agents and the new operator are bundled in a new core component called `telemetry`. The existing Kyma backends and UIs will be just one possible solution to integrate with. They continue to be installable manually by the user (via a blueprint), but will not be part of the actual Kyma offering anymore.
+
+![b](./assets/future.drawio.svg)
+
+As mentioned before, the technology and protocols for the signal collection are different for the signal types. Logs are getting tailed from container log files, metrics are getting pulled via the prometheus format usually and traces are getting pushed via OTLP. With that, also the pre-integration so that typical signals are getting collected instantly are different per type.
+That's why the concrete concepts for the different types are different and will be discussed in more detail in the individual sections:
+* [Concept - Configurable Logging](./configurable-logging/README.md) 
+* [Concept - Configurable Monitoring](./configurable-monitoring/README.md)
+* [Concept - Configurable Tracing](./configurable-tracing/README.md)
+
+All concepts will follow general rules and will provide harmonized user APIs:
+- **By default signals of user workload will be collected**: When a workload gets instrumented as outlined in the best practice, the telemetry data will be instantly available in a pipeline without further configuration, just define a pipeline with an output and it works
```

Comment:
```suggestion
- **By default, signals of user workloads are collected**: If a workload is instrumented as outlined in the best practice guide, the telemetry data is instantly available in a pipeline without further configuration. Just define a pipeline with an output and it works.
```

What's the difference between "guideline", "best practice (guide)", and "blueprint"? Are they synonyms for us or do we differentiate here? If they're used as synonyms, we should settle for just one term and drop the others to improve consistency and simplicity.